project/src/components/place-card-gallery/place-card-gallery.tsx

```diff
@@ -0,0 +1,29 @@
+import { OfferType } from '../../types/offer';
+
+type PlaceCardGalleryProps = {
+  offer: OfferType;
+};
+
+const MAX_PHOTOS = 6;
+
+function PlaceCardGallery({offer}: PlaceCardGalleryProps): JSX.Element {
+
+  return (
+    <div className="property__gallery-container container">
+      <div className="property__gallery">
+        {
+          offer.images.map((imageUrl: string , id: number) => {
+            const keyValue = id + imageUrl;
+            return id < MAX_PHOTOS ? (
```

Comment:
```suggestion
             (
```
отрешь нужное количество фоток с помощью метода `slice`